Rakefile

```diff
@@ -19,6 +19,10 @@ Rake::Task[:mruby].invoke unless Dir.exist?(mruby_root)
 Dir.chdir(mruby_root)
 load "#{mruby_root}/Rakefile"
 
+load "#{mruby_root}/../mrbgem.rake"
```

Comment:
I _think_ it's safe to assume that our `mrbgem.rake` will always be next to `Rakefile`, while `mruby_root` may not always be one directory down from our source dir.